ci/azure-linux-container.yml

```diff
@@ -3,25 +3,29 @@ parameters:
   manylinux: ''
   path: ''
   target: ''
+  verifyManylinux: true
 
 steps:
 - bash: curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly --profile minimal -y
   displayName: rustup
 - bash: PATH=$(path) rustup default nightly
   displayName: ensure nightly
-- bash: PATH=$(path) pip install --user --upgrade pip wheel maturin twine
+- bash: PATH=$(path) $(interpreter) -m pip install --user --upgrade pip wheel maturin twine
   displayName: build dependencies
-- bash: PATH=$(path) pip install --user -r test/requirements.txt -r integration/requirements.txt
+- bash: PATH=$(path) $(interpreter) -m pip install --user -r test/requirements.txt -r integration/requirements.txt
   displayName: test dependencies
 - bash: PATH=$(path) RUSTFLAGS="-Zmutable-noalias -C target-feature=+sse2" maturin build --release --strip --manylinux $(manylinux) --interpreter $(interpreter) --target $(target)
   displayName: build
-- bash: PATH=$(path) pip install --user target/wheels/orjson*.whl
+- bash: PATH=$(path) $(interpreter) -m pip install --user --upgrade auditwheel && PATH=$(path) $(interpreter) -m auditwheel repair target/wheels/orjson*.whl
```

Comment:
I've removed the install of wheel so this could now be installed with pip and the other build tools.

Reply:
`wheel` is already installed in all environments. `auditwheel` is available globally (`/usr/local/bin/`) via Python 3.7, so it doesn't have to be installed. The `pip install --upgrade ... wheel` leads to wheel being upgraded to a version not compatible with `auditwheel` atm. This is only relevant for the Python 3.7 build. The later install of auditwheel ensures that `wheel` is downgraded to the appropriate version.

Idk an idiomatic solution for this. So your solution seems fine, personally, I would install auditwheel directly before the `auditwheel` command in order to avoid any other installs messing with dependencies. Although both awkwardly overwrite the version shipped with manylinux. IMO, auditwheel should be installed to a venv in the manylinux images.